guide/how-it-works/ecash/introduction.md

```diff
@@ -0,0 +1,242 @@
+{% include picture.html
+   image = "/assets/images/guide/how-it-works/ecash/creation-and-redemption.png"
+   retina = "/assets/images/guide/how-it-works/ecash/creation-and-redemption@2x.png"
+   mobile = "/assets/images/guide/how-it-works/ecash/creation-and-redemption@2x.png"
+   mobileRetina = "/assets/images/guide/how-it-works/ecash/creation-and-redemption@2x.png"
+   alt-text = "Diagram illustrating the creation and redemption of ecash by Alice. On the left side, under ‘Creation,’ Alice sends 100 sats to an ecash mint, which creates ecash tokens, and Alice receives 100 sats in ecash form. On the right side, under ‘Redemption,’ Alice sends 100 sats in ecash form to the mint, which burns the ecash tokens, and Alice receives 100 sats back in Bitcoin. The process flow is represented with dashed arrows and labeled steps."
+   caption = "Alice mints 100 sats into ecash through a mint and later redeems the ecash tokens to reclaim her original 100 sats in bitcoin."
+   width = 800
+   height = 508
+   layout = "full-width"
+%}
+
+Cashu and Fedimint use various terms for the processes of creating, redeeming, and transferring bitcoin-backed ecash. To provide clarity, we'll use neutral terms in our explanations and provide a translation table for specific implementations.
+
+#### Creating ecash
+
+**1. User deposits funds** - The user deposits bitcoin into a mint or federation. Typically this is done by generating a lightning invoice through the mint and paying it.
+
+**2. Ecash creation** - Upon successful payment, the wallet generates secrets and [blinds](https://en.wikipedia.org/wiki/Blinding_(cryptography)) them. The wallet then sends the blinded messages to the mint or federation, which returns [blind signature](https://en.wikipedia.org/wiki/Blind_signature) on the blinded messages. The blinding process ensures that the mint cannot link the tokens to the user, preserving privacy.
+
+**3. Ecash receipt** - The user receives a set of proofs that correspond to specific amounts of bitcoin. These proofs can be combined to create an ecash token of any amount denominated in bitcoin.
+
+#### Converting ecash to bitcoin
+
+**1. User initiates conversion** - User initiates a conversion request with their mint or federation.
+
+**2. Ecash verification and invalidation** - The mint or federation verifies the validity of the ecash tokens. Upon verification, the mint or federation invalidates these tokens, ensuring they cannot be reused. This step prevents double-spending.
+
+**3. Bitcoin transfer** - Once the tokens are verified and invalidated, the mint or federation sends an equivalent amount of bitcoin to the user. 
+
+#### Terminology translation table
+
+| Generic term | Cashu term | Fedimint term |
+|-------|----------|----------|
+| Create ecash | Mint | Issue |
+| Send ecash | Swap | Reissue | 
+| Convert ecash to bitcoin | Melt | Redeem |
+| Token | Token | Note |
+
+### Sending and receiving ecash
+{% include picture.html
```

Comment:
How about adding a subtle grey background to the image? Could help with visual page structure. For example, the Title and the caption are quite detached from the image. 

![image](https://github.com/user-attachments/assets/6d3f15b5-e628-42fc-916c-572c812e8aa4)

Reply:
I used to have a subtle gray background, but I removed it. I did this for all the images / illustrations in the ecash section. I prefer the look of everything blending seamlessly into the white background, making it feel unified with the page. To make the title and caption feel more connected to the image, I adjusted the height so they’re closer to the illustration.

<img width="948" alt="image" src="https://github.com/user-attachments/assets/6ea4c008-91aa-48f2-b6e8-c5a787cd5dcf">